data/sql/updates/pending_db_world/rev_1571086874024573263.sql

```diff
@@ -0,0 +1,338 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1571086874024573263');
+
+-- K3 Perimeter Turret: Shoot when the victim is in range between 0-20 yards
+UPDATE `smart_scripts` SET `event_param1` = 0, `comment` = 'K3 Perimeter Turret - Victim In Range 0-20 Yards - Cast ''Energy Surge''' WHERE `entryorguid` = 29483 AND `source_type` = 0 AND `id` = 0;
+
+-- Improved Land Mine: Detonate if the victim is in range between 0-10 yards
+DELETE FROM `smart_scripts` WHERE `source_type` = 0 AND `entryorguid` = 29475;
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`)
+VALUES
+(29475,0,0,1,9,0,100,0,0,10,250,250,0,11,54537,0,0,0,0,0,2,0,0,0,0,0,0,0,0,'Improved Land Mine - Victim In Range 0-10 Yards - Cast Spell Detonation'),
+(29475,0,1,0,61,0,100,0,0,0,0,0,0,41,500,0,0,0,0,0,1,0,0,0,0,0,0,0,0,'Improved Land Mine - Linked - Despawn'),
+(29475,0,2,0,5,0,100,0,0,0,0,29618,0,33,29618,0,0,0,0,0,1,0,0,0,0,0,0,0,0,'Improved Land Mine - On Kill - Kill Credit'),
+(29475,0,3,0,5,0,100,0,0,0,0,29619,0,33,29618,0,0,0,0,0,1,0,0,0,0,0,0,0,0,'Improved Land Mine - On Kill - Kill Credit');
+
+-- Garm Invader: Set active after respawn; start random waypoint movement after 1-60 seconds;
+-- disable auto attack / combat movement while following the waypoint path
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 29619;
+DELETE FROM `smart_scripts` WHERE `source_type` = 0 AND `entryorguid` = 29619;
+DELETE FROM `smart_scripts` WHERE `source_type` = 9 AND `entryorguid` BETWEEN 2961900 AND 2961904;
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`)
+VALUES
```

Comment:
put "VALUES" at the end of the columns list

Reply:
Why?

Reply:
Code style and guidelines on AC wiki http://www.azerothcore.org/wiki/Dealing-with-SQL-files
Indeed I usually see in that style on web, is a sort of standard (or standard de facto).

Reply:
Just to make this clear: I did all the structuring in the SQL script on purpose to improve readability. I also put the "VALUES" in a separate line to act as a separator to the blocks. Those scripts are hard enough to read already, no need to make it more difficult.